internal/controllers/reconciliation/controller.go

```diff
@@ -139,7 +140,7 @@ func (c *Controller) Reconcile(ctx context.Context, req *reconstitution.Request)
 	slice := &apiv1.ResourceSlice{}
 	err = c.client.Get(ctx, req.Manifest.Slice, slice)
 	if err != nil {
-		return ctrl.Result{}, fmt.Errorf("getting resource slice: %w", err)
+		return ctrl.Result{}, client.IgnoreNotFound(fmt.Errorf("getting resource slice: %w", err))
```

Comment:
same as the other case - this should have existed from the beginning to avoid leaking work items when slices are deleted while resources remain in cache